content/en/user-manual/designer/editor-api.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Editor API
+template: usermanual-page.tmpl.html
+position: 11
+---
+
+<div class="alert alert-info">
+The Editor API is a beta feature. Please use caution when using on live projects.
+</div>
+
+The Editor has an user accessible API that is currently in beta which can be used to help automate and extend the base functionality.
```

Comment:
```suggestion
The Editor has a user accessible API that is currently in beta which can be used to help automate and extend the base functionality.
```

Reply:
English grammar question: Is it not 'an' because of the vowel in 'user'?

Reply:
Languages should have consistency 😅  Its the same with 'its' vs 'it's'